### PR TITLE
Cleaned history

### PR DESCRIPTION
**Pull request by sleibman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,119 @@
-# exposure-notifications-metrics
-Aggregate EN metrics into one place
+# COVID-19 Exposure Notification Apps - Metrics Aggregation Tools
+
```
Comment (parkmichelle) [inline README.md:2]: Add the following attribution in normal print + maybe in italic

Contributors: Nat Hillard, Steve Leibman, Chris Pimlott, Michelle Park, Janell Schafer, Kelly Taylor

File: google_workflow/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+   ```sh
+   curl -X POST -H 'Content-Type: application/json'  -d "$(cat data.json| jq '.statistics')" localhost:8080
+   ```
+
+## Deploy
+
+1. Create a secret called `ENCV_api_key` in [secret manager](https://console.cloud.google.com/security/secret-manager)
+2. Create a BigQuery dataset called `encv` in the [BigQuery Console](https://console.cloud.google.com/bigquery).
+3. Deploy the cloud function with `gcloud functions deploy encv_to_db --runtime python38 --trigger-http --allow-unauthenticated --timeout=240`
+4. Deploy the workflow with gcloud workflows deploy co-metrics-workflow --source=workflow.yaml
```
Comment (parkmichelle) [inline google_workflow/README.md:55]: Just checking, is this still "co-metrics-workflow"  or would they run something different?
Same as past questions, fine if this is an example but want to check

Comment (parkmichelle) [reply google_workflow/README.md:55]: same thing on line 56 and 57

File: statemachine/metrics.asl.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
+    "Comment": "A metrics pipeline for gathering data from ",
```
Comment (parkmichelle) [inline statemachine/metrics.asl.json:2]: Was this supposed to be a complete sentence

File: template.yaml [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    Type: AWS::SecretsManager::Secret
+    Properties:
+      Name: "prod/co_metrics/api_keys/encv"
+      Description: The API key for the ENCV API
+      SecretString: !Sub '{"key": "${ENCVAPIKey}"}'
+
+  GOOGLECREDENTIALS:
+    Type: AWS::SecretsManager::Secret
+    Properties:
+      Name: "prod/co_metrics/service_account_credentials"
```
Comment (parkmichelle) [inline template.yaml:85]: probably ok, but is co_metrics here still applicable?

**Review by parkmichelle — COMMENTED** [review]
@sleibman I screened through all the files here again and left comments from an overcautious view, so feel free to take a final pass through! Also added some small documentation updates while I was at it

File: README.md [hunk 5]
```diff
@@ -1,2 +1,119 @@
...
+1. A file named `service.json` needs to be saved as `<top_level>/functions/json_to_sheets/service.json`. This should contain the credentials for the service account associated with the drive/sheets API, and can downloaded from the Google console by: 1. Getting added as an `owner` to the associated Google Developer Console project. 2. going to IAM Admin in the Google console here: https://console.developers.google.com/iam-admin/serviceaccounts , then selecting your project, and going to `“…” -> “Create key”`, or by using an already-generated key (you can only generate a given key once).
+
+
+### Gotchas
+
+1. Rate limiting
+   We've seen the permission-setting portion of the test code cause issues with rate limiting. If you get a message like the following:
+
+   ```sh
+   googleapiclient.errors.HttpError: <HttpError 403 when requesting https://www.googleapis.com/drive/v3/files/1InnwXH0GBBdJkhAKrg_t0woI5Ol5796jwZKc8ANdPYI/permissions?alt=json returned "Rate limit exceeded. User message: "Sorry, you have exceeded your sharing quota."". Details: "Rate limit exceeded. User message: "Sorry, you have exceeded your sharing quota."">
```
[the remark this answers is not in the document]
Comment (sleibman) [reply README.md]: edited.

File: example_azure_pipelines.yml [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+    arguments: ''
+    scriptType: 'inline'
+    inlineScript: |
+      cd encv
+      echo "run make - performing sam build..."
+      make clean
+      make build
+      echo "performing sam package..."
+      cat .aws-sam/build.toml >> sam.toml
+      sam package --config-file sam.toml --s3-bucket cdphe-covidpython-t-release --force-upload --template-file .aws-sam/build/template.yaml --region us-east-1 --output-template-file cdphe-covidpython-t-release.yaml
```
[the remark this answers is not in the document]
Comment (sleibman) [reply example_azure_pipelines.yml]: edited.

**Pull request merged**